charts/newrelic-prometheus-agent/values.yaml

```diff
@@ -182,8 +182,21 @@ config:
   # -- It allows defining scrape jobs for Kubernetes in a simple way.
   # @default -- See `values.yaml`
   kubernetes:
+
+    # By default, all targets discovered are scraped.
```

Comment:
This line could be misleading I think. I would remove it or add more details, like by default targets are discovered based on the `kubernetes-job` filters, having prometheus.io ... blabla

Reply:
I thought I was covering it by saying "targets discovered", I'll try to reformulate the sentence